sonic-xcvrd/xcvrd/xcvrd_utilities/y_cable_helper.py

```diff
@@ -468,6 +489,530 @@ def delete_ports_status_for_y_cable():
+                 ('Nic_Lane1_Precursor2', str(mux_static_info_dict["Nic_Lane1_Precursor2"])),
+                 ('Nic_Lane1_Maincursor', str(mux_static_info_dict["Nic_Lane1_Maincursor"])),
+                 ('Nic_Lane1_Precursor1', str(mux_static_info_dict["Nic_Lane1_Postcursor1"])),
+                 ('Nic_Lane1_Postcursor2', str(mux_static_info_dict["Nic_Lane1_Postcursor2"])),
+                 ('Nic_Lane2_Precursor1', str(mux_static_info_dict["Nic_Lane2_Precursor1"])),
+                 ('Nic_Lane2_Precursor2', str(mux_static_info_dict["Nic_Lane2_Precursor2"])),
+                 ('Nic_Lane2_Maincursor', str(mux_static_info_dict["Nic_Lane2_Maincursor"])),
+                 ('Nic_Lane2_Precursor1', str(mux_static_info_dict["Nic_Lane2_Postcursor1"])),
+                 ('Nic_Lane2_Postcursor2', str(mux_static_info_dict["Nic_Lane2_Postcursor2"])),
+                 ('TOR_self_Lane1_Precursor1', str(mux_static_info_dict["TOR_self_Lane1_Precursor1"])),
+                 ('TOR_self_Lane1_Precursor2', str(mux_static_info_dict["TOR_self_Lane1_Precursor2"])),
+                 ('TOR_self_Lane1_Maincursor', str(mux_static_info_dict["TOR_self_Lane1_Maincursor"])),
+                 ('TOR_self_Lane1_Precursor1', str(mux_static_info_dict["TOR_self_Lane1_Postcursor1"])),
+                 ('TOR_self_Lane1_Postcursor2', str(mux_static_info_dict["TOR_self_Lane1_Postcursor2"])),
+                 ('TOR_self_Lane2_Precursor1', str(mux_static_info_dict["TOR_self_Lane2_Precursor1"])),
+                 ('TOR_self_Lane2_Precursor2', str(mux_static_info_dict["TOR_self_Lane2_Precursor2"])),
+                 ('TOR_self_Lane2_Maincursor', str(mux_static_info_dict["TOR_self_Lane2_Maincursor"])),
+                 ('TOR_self_Lane2_Precursor1', str(mux_static_info_dict["TOR_self_Lane2_Postcursor1"])),
+                 ('TOR_self_Lane2_Postcursor2', str(mux_static_info_dict["TOR_self_Lane2_Postcursor2"])),
+                 ('TOR_peer_Lane1_Precursor1', str(mux_static_info_dict["TOR_peer_Lane1_Precursor1"])),
+                 ('TOR_peer_Lane1_Precursor2', str(mux_static_info_dict["TOR_peer_Lane1_Precursor2"])),
+                 ('TOR_peer_Lane1_Maincursor', str(mux_static_info_dict["TOR_peer_Lane1_Maincursor"])),
+                 ('TOR_peer_Lane1_Precursor1', str(mux_static_info_dict["TOR_peer_Lane1_Postcursor1"])),
+                 ('TOR_peer_Lane1_Postcursor2', str(mux_static_info_dict["TOR_peer_Lane1_Postcursor2"])),
+                 ('TOR_peer_Lane2_Precursor1', str(mux_static_info_dict["TOR_peer_Lane2_Precursor1"])),
+                 ('TOR_peer_Lane2_Precursor2', str(mux_static_info_dict["TOR_peer_Lane2_Precursor2"])),
+                 ('TOR_peer_Lane2_Maincursor', str(mux_static_info_dict["TOR_peer_Lane2_Maincursor"])),
+                 ('TOR_peer_Lane2_Precursor1', str(mux_static_info_dict["TOR_peer_Lane2_Postcursor1"])),
+                 ('TOR_peer_Lane2_Postcursor2', str(mux_static_info_dict["TOR_peer_Lane2_Postcursor2"]))
```

Comment:
Can we stay consistent with the use of case? Let's make the keys all lowercase, e.g., `nic_lane1_precursor1`, `tor_self_lane2_postcursor2`, etc.

Reply:
fixed